.goreleaser.yml

```diff
@@ -46,3 +45,66 @@ dockers:
   - "--label=org.opencontainers.image.version={{.Version}}"
   - "--label=org.opencontainers.image.source={{.GitURL}}"
   - "--platform=linux/amd64"
+- image_templates:
+  - 'skibish/ddns:{{ .Tag }}-arm64'
```

Comment:
Let's add here template for `:latest-arm64` and the same for other builds. This then will build `:latest` and `:{{.Tag}}` versions, and will push them